### PR TITLE
feat: (Core) introduce new Object Identifier

### PR DESCRIPTION
**Pull request by JKMarkowski**
#### Please provide a link to the associated issue.
closes https://github.com/SAP/fundamental-ngx/issues/3425
#### Please provide a brief summary of this pull request.
There is included `ObjectIdentifier` component
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Review by InnaAtanasova — COMMENTED** [review]
Could you add an example with no description?
I think it's a good idea to copy the documentation part from fund-styles and if needed to add ngx specific stuff. We should be unifying our documentations. 
The rest looks good

File: apps/docs/src/app/core/component-docs/object-identifier/object-identifier-docs.component.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<fd-docs-section-title [id]="'default'" [componentName]="'objectIdentifier'">
```
Comment (Betrozov) [inline apps/docs/src/app/core/component-docs/object-identifier/object-identifier-docs.component.html:1]: unnecessary binding
plz check the whole file

File: libs/core/src/lib/object-identifier/object-identifier.component.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+} from '@angular/core';
+import { LinkComponent } from '../link/link.component';
+import { startWith, takeUntil } from 'rxjs/operators';
+import { Subject } from 'rxjs';
+
+@Component({
+    selector: 'fd-object-identifier',
+    template: `
+        <p class="fd-object-identifier__title"
+           [ngClass]="{'fd-object-identifier__title--bold': bold}">
```
Comment (Betrozov) [inline libs/core/src/lib/object-identifier/object-identifier.component.ts:23]: [class.fd-object-identifier__title--bold]="bold"

File: libs/core/src/lib/object-identifier/object-identifier.component.ts [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    ) {}
+
+    /** @hidden */
+    ngAfterContentInit(): void {
+        this._listenOnLinkQueryChange();
+    }
+
+    /** @hidden */
+    ngOnChanges(changes: SimpleChanges): void {
+        if (changes && changes['large']) {
```
Comment (Betrozov) [inline libs/core/src/lib/object-identifier/object-identifier.component.ts:72]: plz move to variable `changes['large']`

**Review by Betrozov — CHANGES_REQUESTED** [review]

File: libs/core/src/lib/object-identifier/object-identifier.component.spec.ts [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    bold: boolean;
+    medium: boolean;
+
+    getTitleElementClassList(): DOMTokenList {
+        const elements = document.getElementsByClassName('fd-object-identifier__title');
+        return elements.item(0).classList;
+    }
+}
+
+fdescribe('ObjectIdentifierComponent', () => {
```
Comment (mikerodonnell89) [inline libs/core/src/lib/object-identifier/object-identifier.component.spec.ts:37]: fdescribe

**Review by mikerodonnell89 — CHANGES_REQUESTED** [review]
Looks great, just remove fdescribe from test

**Comment by JKMarkowski** [comment]
@InnaAtanasova , @mikerodonnell89, @Betrozov, Thanks for taking a look :) You comments has been addressed, so it's ready for rereview

**Review by Betrozov — APPROVED** [review]

**Review by InnaAtanasova — APPROVED** [review]

**Review by mikerodonnell89 — APPROVED** [review]

**Pull request merged**